packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -185,7 +186,7 @@ export const SelectTree = function SelectTree( {
 					) : (
 						<TextControl
 							{ ...inputProps }
-							value={ props.createValue || '' }
+							value={ decodeEntities( props.createValue || '' ) }
```

Comment:
Are we using the single selection tree anywhere so I can test this? I'm wondering if it is needed (same as above comment).

Reply:
Yes, this part is used in the field "Parent" after an entry is selected